README.md

```diff
@@ -10,6 +10,19 @@
 
 The majority of ADI's products are peripherals to a non-ADI digital engine (FPGA, microprocessor, or microcontroller). While there is major work underway on FPGAs (Xilinx and Intel/Altera) and microprocessors (running an operating system like Linux), the efforts on microcontrollers are fragmented due to the diverse nature of the microcontroller market. The goal of these projects (microcontroller/no-OS) is to be able to provide reference projects for lower end processors, which can't run Linux or aren't running a specific operating system, to help customers using FPGA/microcontrollers with ADI parts.
 
+## Documentation
+
+Drivers and projects [code documentation](http://analogdevicesinc.github.io/no-OS/).
```

Comment:
I would make it clear that this is only the Doxygen documentation. We should add a link to wiki.analog.com too.